jbot/src/main/java/me/ramswaroop/jbot/core/slack/models/Event.java

```diff
@@ -21,6 +21,9 @@
 
     private int id;
     private String type;
+    private String subtype;
+    @JsonProperty("bot_id")
+    private String botId;
```

Comment:
@ramswaroop botId will part of your message event, when the subtype of message "bot_message". please refer [this link](https://api.slack.com/events/message/bot_message)